build/release/Makefile

```
@@ -121,7 +121,7 @@ promote.helm: $(HELM)
 #	copy existing charts to a temp dir, then combine with new charts, reindex, and upload
 	$(S3_SYNC) s3://$(HELM_S3_BUCKET)/$(HELM_CHANNEL) $(HELM_TEMP)
 	$(S3_SYNC) s3://$(S3_BUCKET)/build/$(BRANCH_NAME)/$(VERSION)/charts $(HELM_TEMP)
-	$(HELM) repo index --url $(HELM_URL) $(HELM_TEMP)
+	$(HELM) repo index --url $(HELM_URL) --merge  $(HELM_TEMP)
```

We need the path to the index.yaml. Running a local test with that command at least did something, though I couldn't fully test a release. Let's try this for now.
```suggestion
	$(HELM) repo index --url $(HELM_URL) --merge $(HELM_TEMP)/index.yaml $(HELM_TEMP)
```